art/attacks/evasion/imperceptible_asr/imperceptible_asr.py

```diff
@@ -58,11 +61,11 @@ class ImperceptibleASR(EvasionAttack):
         "batch_size",
     ]
 
-    _estimator_requirements = (TensorFlowV2Estimator, SpeechRecognizerMixin)
+    _estimator_requirements = (NeuralNetworkMixin, LossGradientsMixin, BaseEstimator, SpeechRecognizerMixin)
 
     def __init__(
         self,
-        estimator: "TensorFlowV2Estimator",
+        estimator: Union["PyTorchEstimator", "TensorFlowV2Estimator"],
```

Comment:
For typing we should create a new type in `art.utils` line 175 and call it "SPEECH_RECOGNIZER_TYPE" and make it a Union of `PyTorchDeepSpeech` and `TensorFlowLingvoAsr`.